src/frontend/data/stores/useLiveReturn/index.ts

```diff
@@ -0,0 +1,6 @@
+import { createStore } from 'utils/createContext';
+
+const store = createStore<boolean>();
+
+export const LiveReturnProvider = store.Provider;
+export const useLiveReturn = store.useStore;
```

Comment:
IMO, using return as part of a name may lead to confusion.
I propose to rename it to something like `LiveFeedbackProvider`, `useLiveFeedback`, etc…
The same applies to all your PR.